uaclient/cli.py

```diff
@@ -145,6 +146,31 @@ def status_parser(parser):
     usage = USAGE_TMPL.format(name=NAME, command="status")
     parser.usage = usage
     parser.prog = "status"
+    # This formatter_class ensures that our formatting below isn't lost
+    parser.formatter_class = argparse.RawDescriptionHelpFormatter
+    parser.description = textwrap.dedent(
+        """\
+        Report current status of Ubuntu Advantage services on system.
+
+        This shows whether this machine is attached to an Ubuntu Advantage
+        support contract. When attached, the report includes the specific
+        support contract details including account and subscription name,
+        expiry dates (if applicable), and the status of each service on this
+        system.
+
+        Each service status line has four columns:
+
+        * SERVICE: name of the service
+        * ENTITLED: whether the contract to which this machine is attached
+          entitles use of this service. Possible values are: yes or no
+        * STATUS: whether the service is enabled on this machine.  Possible
+          values are: enabled, disabled, or — (if inapplicable)
+        * DESCRIPTION: a brief description of the service
```

Comment:
Should we clarify that this set of columns applies to an attached machine (and also add a section for the unattached machine which surfaces the "AVAILABLE" column indicating that a particular service is potentially available on the current series.


Since SERVICE and DESCRIPTION are the same on unattached, maybe we just include a line about AVAILABLE in this existing table and note that AVAILABLE is unattached only and STATUS/ENTITLED are on attached machines.